packages/timedesc-tzlocal-js/timedesc-tzlocal-js.0.8.0/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "JS implementation for timedesc-tzlocal"
+description: ""
+maintainer: ["Darren Ldl <darrenldldev@gmail.com>"]
+authors: ["Daypack developers"]
+license: "MIT"
+homepage: "https://github.com/daypack-dev/timere"
+bug-reports: "https://github.com/daypack-dev/timere/issues"
+depends: [
+  "ocaml" {>= "4.08.1"}
+  "dune" {>= "2.9"}
+  "timedesc-tzlocal" {= "0.8.0"}
```

Comment:
For next time, know that you can replace this by this to make it more clear it comes from the same project and make maintenance easier
```suggestion
  "timedesc-tzlocal" {= version}
```

Reply:
Oh that's very neat! I'll push a fix for this. Thanks!

Reply:
Okay found section on interpolation within string via `%{...}%`